helix-term/src/keymap/default.rs

```diff
@@ -179,6 +179,10 @@ pub fn default() -> HashMap<Mode, Keymap> {
             "C-j" | "j" | "down" => jump_view_down,
             "C-k" | "k" | "up" => jump_view_up,
             "C-l" | "l" | "right" => jump_view_right,
+            "L" => swap_view_right,
+            "K" => swap_view_up,
+            "H" => swap_view_left,
+            "J" => swap_view_down,
```

Comment:
This will contradict with kakoune keys as in uppercase have different meanings but I think it is fine.